index.js

```diff
@@ -1,8 +1,8 @@
 export default function isPlainObject(value) {
-	if (Object.prototype.toString.call(value) !== '[object Object]') {
+	if (typeof value !== 'object' || value === null) {
 		return false;
 	}
 
 	const prototype = Object.getPrototypeOf(value);
-	return prototype === null || prototype === Object.prototype;
+	return (prototype === null || prototype === Object.prototype) && !(Symbol.toStringTag in value) && !(Symbol.iterator in value);
```

Comment:
See https://github.com/sindresorhus/is-plain-obj/issues/12#issuecomment-1154263604